cogs/squid.py

```diff
@@ -0,0 +1,38 @@
+# imports to make the dream
+
+import random
+import asyncio
+import discord
+import commands
+
+''' dab '''
+class dab:
+    def __init__(self, bot):
+        self.bot = bot
+    
+    @commands.command(name = "squid", pass_context = True)
```

Comment:
PEP8: Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.